wireguard-install.sh

```diff
@@ -174,14 +174,14 @@ if [[ $1 == "add-client" ]]; then
 		exit 1
 	fi
 elif [[ $1 == "uninstall" ]]; then
-	if [[ -e /etc/wireguard ]]; then
+	if [[ -e /etc/wireguard/params ]]; then
```

Comment:
The uninstall path should remain `/etc/wireguard`: if the user previously installs wireguard, so `/etc/wireguard/params` doesn't exists, and wants to do a clean reinstallation he can use this option to clean the configuration folders and reinstall it. You should be able to uninstall it even if it's not configured by this script.

From https://github.com/angristan/wireguard-install/issues/80#issuecomment-635214132
> Maybe the error should be ERROR: It seems you have already configured WireGuard, so this script will not run. To use this script, please delete or rename the directory /etc/wireguard and run this script again. In case you have run this script successfully before, it is now time to add clients by passing the add-client argument to this script. If that is the case, leave the contents of /etc/wireguard alone.